src/payment/__factories__/basket.factory.js

```diff
@@ -4,7 +4,7 @@ import Chance from 'chance'; // eslint-disable-line import/no-extraneous-depende
 import './product.factory';
 import './message.factory';
 
-const chance = new Chance();
```

Comment:
We're using the output of chance in snapshots now, so I'm seeding the RNG so it's always the same.